paimon-format/src/main/java/org/apache/paimon/format/orc/OrcFileFormat.java

```diff
@@ -71,13 +74,29 @@ public class OrcFileFormat extends FileFormat {
     private final int writeBatchSize;
     private final boolean deletionVectorsEnabled;
 
+    private static final org.apache.hadoop.conf.Configuration emptyConf =
+            new org.apache.hadoop.conf.Configuration();
+    private static final ObjectCacheManager<Properties, Configuration> configCache =
```

Comment:
Just use `Cache`, `ObjectCacheManager` is too simple, we don't need a class to wrap Cache.